xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -424,10 +426,14 @@ bool Win32DllLoader::ResolveOrdinal(const char *dllName, unsigned long ordinal,
 
 extern "C" FARPROC __stdcall dllWin32GetProcAddress(HMODULE hModule, LPCSTR function)
 {
-  // first check whether this function is one of the ones we need to wrap
-  void *fixup = NULL;
-  if (FunctionNeedsWrapping(win32_exports, function, &fixup))
-    return (FARPROC)fixup;
+  // if the high-order word is zero, then lpProcName is the function's ordinal value
+  if (reinterpret_cast<uintptr_t>(function) > std::numeric_limits<WORD>::max())
```

Comment:
Meh I'm getting rusty in my win32 api skills, this operation is common enough that there's a macro for it :) https://msdn.microsoft.com/en-us/library/windows/desktop/ms632657(v=vs.85).aspx